components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -290,19 +344,21 @@ export default {
     }
 
     &--citation-links {
-      font-weight: bold;
-      font-size: 14px;
-      margin-top: 1rem;
-      a {
-        text-decoration: none;
-        color: $median;
-        font-size: 14px;
-        font-weight: 500;
-        line-height: 16px;
-        cursor: pointer;
-        &.active-citation {
+      border-bottom: 1px solid black;
+      display: flex;
+      list-style: none;
```

Comment:
On mobile, the citation links don't break as well as they could. Adding this allows it wrap as needed, which resolves the issue.

# Before
![localhost_3000_datasets_79_type=dataset(Pixel 2)](https://user-images.githubusercontent.com/1493195/85323925-eea36080-b496-11ea-911e-4d30c2d40afd.png)

# After
![localhost_3000_datasets_79_type=dataset(Pixel 2) (1)](https://user-images.githubusercontent.com/1493195/85324034-17c3f100-b497-11ea-8f6c-465a1bae55b2.png)
```suggestion
      border-bottom: 1px solid black;
      display: flex;
      flex-wrap: wrap;
      list-style: none;
```

Reply:
Ah, forgot about mobile. Good catch, I'll update the style!